iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/vector/HiveVectorizedReader.java

```diff
@@ -253,7 +254,7 @@ private static RecordReader<NullWritable, VectorizedRowBatch> parquetRecordReade
 
   private static CloseableIterable<HiveBatchContext> createVectorizedRowBatchIterable(
       RecordReader<NullWritable, VectorizedRowBatch> hiveRecordReader, JobConf job, int[] partitionColIndices,
-      Object[] partitionValues, Map<Integer, ?> idToConstant) {
+      Object[] partitionValues, Map<Integer, ?> idToConstant, PartitionSpec partitionSpec) {
```

Comment:
is that a leftover? where do we use `partitionSpec`? wasn't it moved to `idToConstant`

Reply:
Removed.